domene/src/main/kotlin/no/nav/tiltakspenger/domene/behandling/Utfallsperiode.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.tiltakspenger.domene.behandling
+
+import java.time.LocalDate
+
+data class Utfallsperiode(
+    val fom: LocalDate,
+    val tom: LocalDate,
+    val antallBarn: Int,
+    val utfall: UtfallForPeriode,
+) {
+    override fun equals(other: Any?): Boolean {
```

Comment:
Hvorfor er ikke dato-feltene med i equals? Jeg antar at det er fordi equals brukes på en spesiell måte i periodiseringslogikken, men er det smart å override equals på denne måten? To perioder med ulike datoer er jo strengt tatt ikke equal...?

Reply:
Litt enig. Vi brukte equals for å kunne shorthande med == i sammenslåingslogikking, men kan kanskje by på uforutsette problemer down the line i situasjoner hvor man bruker likhet, mengder, feks. Foreslår at vi lager vår egen fun i stedet, feks kanSlåsSammen().